repositories/earRepository.go

```diff
@@ -0,0 +1,190 @@
+package repositories
+
+import (
+	"context"
+	"errors"
+	"internship_project/models"
+	"internship_project/persistence"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	uuid "github.com/satori/go.uuid"
+)
+
+type EarRepository struct {
```

Comment:
I suggest using full or at least a little bit shortened names for every entity, especially in the ddl, because these acronyms can be confusing.